compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -258,7 +258,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                         let val = layout.offset_of_subfield(self, fields.iter()).bytes();
                         Scalar::from_target_usize(val, self)
                     }
-                    mir::NullOp::UbChecks => Scalar::from_bool(self.tcx.sess.opts.debug_assertions),
+                    mir::NullOp::UbChecks => Scalar::from_bool(self.tcx.sess.ub_checks()),
```

Comment:
Should this code in `rustc_middle` also be updated? https://github.com/rust-lang/rust/blob/9d79cd5f79e75bd0d2083260271307ce9acd9081/compiler/rustc_middle/src/mir/mod.rs#L802-L803

Reply:
:facepalm: YES